### ISSUE TITLE
Could you give more remarks on DxgkddiInterruptRoutine MessageNumber?

### ISSUE THREAD

**Issue by coffeehat** [MicrosoftDocs/windows-driver-docs-ddi#1471]
Hi,

Doc for [`DxgkddiInterruptRoutine`](https://learn.microsoft.com/en-us/windows-hardware/drivers/ddi/dispmprt/nc-dispmprt-dxgkddi_interrupt_routine) is good. However, I'm still confused on its parameter `MessageNumber`. I can understand that for line-based interrupts, `MessageNumber` should always be zero. But I'm confused what `MessageNumber` could be under message-signaled interrupt, the doc doesn't refer to this.

I guess that `MessageNumber` could be the interrupt index (start from zero) for GPU devices who have multiple MSI. I'm not sure if I am correct.

So I wonder if you can add some remarks or examples on `MessageNumber` under MSI? Thank you

**Comment by lorihollasch** [comment]
@coffeehat, you are correct that MessageNumber is the zero-based index number in the MSI table. I've updated the page accordingly. Thanks for filing an issue!
